src/edu/washington/escience/myria/api/encoding/CreateIndexEncoding.java

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ */
+package edu.washington.escience.myria.api.encoding;
+
+import java.util.List;
+
+import edu.washington.escience.myria.RelationKey;
+import edu.washington.escience.myria.Schema;
+import edu.washington.escience.myria.accessmethod.AccessMethod.IndexRef;
+
+/**
+ * 
+ */
+public class CreateIndexEncoding extends MyriaApiEncoding {
+  @Required
+  public RelationKey relationKey;
+  @Required
+  public Schema schema;
```

Comment:
I question the necessity of including the schema. Isn't this redundant information since, given `relationKey`, we can look up the schema from the catalog, and infer the columns that should be indexed from the offsets in `indexes`?